aea/manager/manager.py

```diff
@@ -366,6 +366,7 @@ def stop_manager(
             self._thread.join(self.DEFAULT_TIMEOUT_FOR_BLOCKING_OPERATIONS)
 
         self._thread = None
+        self._warning_message_printed_for_agent = {}
```

Comment:
I feel this was missing? Whenever we stop the multiagent manager, we want that the next time it is run, the warning messages are handled again